contracts/scripts/deploy.ts

```diff
@@ -0,0 +1,27 @@
+import { ethers } from "hardhat";
+
+async function main() {
```

Comment:
We could also maybe move the deploy/mint tasks over from bridge to here too, so the CI seed script can just use this repo. Useful to have a way to easily deploy contracts with any name/symbol/decimals

https://github.com/Kava-Labs/kava-bridge/blob/main/contract/tasks/deployERC20.ts

Reply:
good idea, it would be nice to remove kava-bridge from this repo's CI. i created a ticket for this work: [sc9364]